spring-cloud-gcp-samples/spring-cloud-gcp-trace-sample/src/test/java/com/example/TraceSampleApplicationIntegrationTests.java

```diff
@@ -82,17 +95,49 @@
 
   private TraceServiceBlockingStub traceServiceStub;
 
-  @BeforeClass
-  public static void checkToRun() {
-    assumeThat(
-        "Google Cloud Trace integration tests are disabled. "
-            + "Please use '-Dit.trace=true' to enable them. ",
-        System.getProperty("it.trace"),
-        is("true"));
+  private static final String SAMPLE_TOPIC = "traceTopic";
+
+  private static final String SAMPLE_SUBSCRIPTION = "traceSubscription";
```

Comment:
Same for subscription; let's have it be unique to reduce flakes when multiple integration tests run simulataneously.